kartothek/core/common_metadata.py

```diff
@@ -380,7 +380,17 @@ def store_schema_metadata(schema, dataset_uuid, store, table):
 
 def _schema2bytes(schema):
     buf = pa.BufferOutputStream()
-    pq.write_metadata(schema, buf, version="2.0", coerce_timestamps="us")
+    # [#259] setting all timestamp fields to us-granularity (parquet default) instead of relying on pq.write_metadata coerce_timestamps="us"  
+    fields = []
+    for idx in range(len(schema)):
```

Comment:
It should be possible to iterate over the schema directly, i.e. 
```
for field in schema:
    assert isinstance(field, pa.Field)
```

Might be only introduced in the latter version of arrow so I'm not sure if this is fully supported since we still support pyarrow==0.13.0